src/tool/subcommands/api_cmd.rs

```diff
@@ -2043,7 +2223,7 @@ async fn run_tests(
                         true
                     }
                     PolicyOnRejected::PassWithQuasiIdenticalError
-                        if reason_lotus.starts_with(reason_forest) =>
+                        if reason_lotus.contains(reason_forest) =>
```

Comment:
This may impact other tests that rely on an error string appearing at the beginning rather than elsewhere.

Reply:
`contains` checks for the `reason_forest` to be present anywhere in `reason_lotus` should work even if `reason_lotus` starts with `reason_forest`